product-scenarios/10-manage-security-resiliancy-other-aspects-of-service-endpoints/10.1-expose-service-with-secured-backend/10.1.1-expose-service-secured-via-basic-authentication/pom.xml

```diff
@@ -70,6 +70,7 @@
         <dependency>
             <groupId>org.wso2.am</groupId>
             <artifactId>org.wso2.carbon.apimgt.samples.utils</artifactId>
+            <version>2.6.1-SNAPSHOT</version>
```

Comment:
${apimserver.version} should be the correct one

Reply:
Resolved